### DISCUSSION TITLE
The :attribute failed to upload - not translated

### DISCUSSION THREAD

**Discussion by gr1ng0wf** [livewire/livewire#3015] [7. Bugs (archived)]
### Description
If file failed upload then output error: `The :attribute failed to upload.`  without translated attribute. I trying define globaly and localy validation attributes and they not working. [Image output validation error](https://imgur.com/jAUPFwA).
I also noticed that in trait WithUploadedFiles in method uploadErrored missing attribute translation after getting error validation message. [Screen method uploadErrored](https://imgur.com/9Fwg4HA)

### Exact steps to reproduce
1. Create upload file form with Livewire
2. Add translate validations attribute from component or globaly validation.php
3. Get large file out of range php settings and fill input file
4. Submit form
5. Check validation error message
_in my case i expect to see the translated attribute_

### Stripped-down, copy-pastable code snippets
-

**Context**
- Livewire version: 2.4
- Laravel version: 8.12
- Alpine version: 2.8
- Browser: Chrome

**Comment by calebporzio** [comment]
I believe a PR was recently submitted for this, can you upgrade Livewire to "dev-master" in your composer.json and check that this is still an issue?
